src/cluster_legacy.c

```diff
@@ -2691,49 +2865,38 @@ static clusterNode *getNodeFromLinkAndMsg(clusterLink *link, clusterMsg *hdr) {
     return sender;
 }
 
-/* When this function is called, there is a packet to process starting
- * at link->rcvbuf. Releasing the buffer is up to the caller, so this
- * function should just handle the higher level stuff of processing the
- * packet, modifying the cluster state if needed.
- *
- * The function returns 1 if the link is still valid after the packet
- * was processed, otherwise 0 if the link was freed since the packet
- * processing lead to some inconsistency error (for instance a PONG
- * received from the wrong sender ID). */
-int clusterProcessPacket(clusterLink *link) {
+bool clusterValidatePacket(clusterLink *link) {
```

Comment:
Surprise!

I haven't seen bool in valkey before. Not sure why. I think we can start using it. Need to include bool.h explicitly.

The function name can be clusterIsValidPacket instead, to make the meaning of the return value more clear.